@here/harp-examples/src/datasource_webtile.ts

```diff
@@ -63,7 +63,8 @@ export namespace WebTileDataSourceExample {
     // snippet:harp_gl_datasource_webtile_1.ts
     const webTileDataSource = new WebTileDataSource({
         appId,
-        appCode
+        appCode,
+        ppi: 320
```

Comment:
this seems to be for a phone, is that the general use case that this example should show?

Reply:
It is what it was showing before, I think for most of the browsers is the best setting